### PR TITLE
clangformat support cuda file

### PR DESCRIPTION
**Pull request by Presburger**

### CODE REVIEW

**Comment by Presburger** [comment]
@chaoren ，please help review the code.

**Comment by chaoren** [comment]
I'm not a maintainer of this repository, so I'm not sure why you're asking me to review this. You'll probably want to sign that CLA in https://github.com/google/vim-codefmt/pull/211#issuecomment-1442877590.

**Comment by Presburger** [comment]
> I'm not a maintainer of this repository, so I'm not sure why you're asking me to review this. You'll probably want to sign that CLA in [#211 (comment)](https://github.com/google/vim-codefmt/pull/211#issuecomment-1442877590) though.

thanks

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ helpfiles in the `doc/` directory. The helpfiles are also available via
 # Supported File-types
 
 * [Bazel](https://www.github.com/bazelbuild/bazel) BUILD files (buildifier)
-* C, C++ (clang-format)
+* C, C++, Cuda (clang-format)
```
Comment (chaoren) [inline README.md:14]: I don't think Cuda needs to be distinct from C/C++ here.

Comment (dbarnett) [reply README.md:14]: Meaning just revert this change, and "C, C++" implies Cuda as well?

Comment (dbarnett) [reply README.md:14]: If it does need to be explicitly listed, note there's also a listing at https://github.com/google/vim-codefmt/blob/1b76987b6719bee259ba8d1e03a1e7f624a3615f/plugin/register.vim#LL30C20-L30C20 you haven't updated, and generated docs based on that list.

Comment (chaoren) [reply README.md:14]: > "C, C++" implies Cuda as well?

Yeah, I think CUDA just adds a few extensions to C/C++.

Comment (dbarnett) [reply README.md:14]: K, so @Presburger if so you could just revert this and keep just the vimscript change and I can merge it.

**Review by dbarnett — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Presburger) [reply README.md]: > K, so @Presburger if so you could just revert this and keep just the vimscript change and I can merge it.

OK, well, Cuda is a extend of C++ & C, i agree with yours suggest.

**Comment by dbarnett** [comment]
Alright, thanks! Merging.

**Review by dbarnett — APPROVED** [review]

**Pull request merged**